doc_git.md

```diff
@@ -0,0 +1,90 @@
+# OUR FLOW :
+
+#### Branch creation:
+
+We always work on our own branch, which will be named in the same fashion : "[initials]/[feature]" == el/my_new_branch  
+To create such a branch  
+	`git checkout -b [initials]/[branch name]`
+
+/!\ No one is allowed to code on this branch, except its owner. If you need to test it or you need some of the code, you **must** create a new branch using the above command.  
```

Comment:
Starting a review to show the process to the others not used to it.

Based on this document: https://datatracker.ietf.org/doc/html/rfc2119

It would make sense to replace the **must** by MUST or even **MUST** to emphasize them and comply with commonly used norms.